### DISCUSSION TITLE
Got approved, but I'm still "Free" tier

### DISCUSSION THREAD

**Discussion by sthefanoss** [community/community#145057] [GitHub Education] [answered]
### Hi everyone,

Hey! Firstly thanks for your time.

![image](https://github.com/user-attachments/assets/a498d44c-c281-4f64-8f53-9cf2e27931d8)
I have got approved but I'm still free tier. 

Have I missed anything?

**Comment by davevad93** [comment]
Hi @sthefanoss ,

Given your status, you should have access to your benefits indeed. Try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select one of the available offers, for example [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

**Reply by sthefanoss** [reply]
@davevad93 worked for GitKraken and Copilot. Thanks!

But, differently from my last application, I'm a "free" user. Is this expected?

![image](https://github.com/user-attachments/assets/8555cec7-32e3-4637-8b40-4ad1767fd486)

**Reply by davevad93** [reply, marked as answer]
You're welcome @sthefanoss , don't worry actually you are a `PRO` user, that `Free` label refers to the free trial of Enterprise.

<img width="269" alt="0000" src="https://github.com/user-attachments/assets/89f0f140-7e88-4037-9fc2-6dae940e09f4">
